web/src/app/app.component.ts

```diff
@@ -41,6 +41,11 @@ export class AppComponent implements AfterViewInit {
       route: '/projects',
       icon: 'project_page_icon',
     },
+    {
+      title: 'Following',
+      route: '/following',
+      icon: 'stats_user_icon',
```

Comment:
I will find a different icon

Reply:
I moved to "My Projects" page at the top right

<img width="1560" alt="Screenshot 2019-10-20 08 31 14" src="https://user-images.githubusercontent.com/624760/67156260-fdd47c00-f313-11e9-9a87-eed0ca1f7ca5.png">